### ISSUE TITLE
icingadb: delivered systemd service is missing postgres dependency

### ISSUE THREAD

**Issue by EmmaTinten** [Icinga/icinga-packaging#256]
The packaged version of the systemd.unit does not depend on postgresql. If you are using postgresql, redis will fail and not start up on boot.

Delivered:

```
# /lib/systemd/system/icingadb.service
[Unit]
Description=Icinga DB
After=syslog.target network-online.target icingadb-redis.service mariadb.service

[Service]
Type=notify
ExecStart=/usr/sbin/icingadb --config /etc/icingadb/config.yml
User=icingadb
Group=icingadb

[Install]
WantedBy=multi-user.target
```

Please add "postgresql.service" on Debian which will fix the problem.

**Comment by julianbrost** [comment]
It's also missing in the RPM packages.

**Comment by htriem** [comment]
This has been fixed with release 1.1.0, so I'll close this.
